README.md

```diff
@@ -31,30 +25,40 @@ The repository consists of:
-    * The resources required to be backed up
-    * A service principal used for deployment, with the following permissions:
-      * Create resources
-      * Assign roles on the backup resources
+* Azure backup vault and backup policies/instances
+* Azure policy definitions and assignments
+* Azure monitor
+* Entra ID
 * Tfstate storage account
-  * This must be created once during environment setup - see Developer Guide below for more details
-* Backup vault and associated policies/instances
-  * These are defined in the terraform scripts in this repo
-  * The scripts should be customised for the specific use case and deployed via a deployment pipeline (or `terraform apply` in dev scenario)
-  * A resource group is created which all backup resources reside within
+* Resources that need to be backed up
 
-![Azure Architecture](./docs/azure-architecture.png)
+#### Architecture
 
-### Pipelines
+The following diagram illustrates the high level architecture
+
+![Azure Architecture](./docs/azure-architecture.drawio.svg)
+
+1. The **backup vault** stores the backups of a variety of different Azure resources. A number of **backup policies** are registered on the vault which define the configuration for a backup such as the retention period and schedule. A number of **backup instances** are then registered with a policy applied that trigger the backups. The vault is configured as **immutable** and **locked** to enforce tamper proof backups. The **backup vault** resides in it's own isolated **resource group**.
+
+1. **Backup instances** link the resources to be backed up and an associated **backup policy**, and one registered trigger the backup process. The resources directly supported are Azure Blob Storage, Managed Disks, PostgreSQL (single server and flexible server) and AKS instances, although other resources are supported indirectly through Azure Storage (see **point 8** for more details). **Backup instances** are automatically registered by **Azure Policy** by creating resources to be  backed up with the required tags - they are not manually registered (see **point 4** for more details).
+
+1. The **backup vault** accesses resources to be backed up through a **System Assigned Managed Identity** - a secure way of enabling communication between defined resources without managing a secret/password. The identity is given read access to the resources to be backed up by **Azure Policy** at the point that the backup instance is registered.
+
+1. **Azure Policy** is an feature that helps enforce rules and standards across an Azure tenant. In this case it is used to ensure **backup instances** are created when resources that require backup are created (or updated) with a defined tag. **Azure Policy** will also be used to validate the **immutability** configuration of the backup vault, for example ensuring it is not set excessively resulting in a developers holiday photos being stored for 100'000 years.
```

Comment:
```suggestion
1. **Azure Policy** is an feature that helps enforce rules and standards across an Azure tenant. In this case it is used to ensure **backup instances** are created when resources that require backup have a defined tag. **Azure Policy** will also be used to validate the **immutability** configuration of the backup vault, for example ensuring it is not set excessively resulting in a developers holiday photos being stored for 100'000 years.
```

Minor tweak here to avoid repetitive use of "created" in the sentence

This question will _probably_ be answered in future iterations of the documentation for this but I was wondering if there would be specific formatting requirements for the defined tags?